src/nav/kubernetes-pixie.yml

```diff
@@ -9,52 +9,50 @@ pages:
             path: /docs/kubernetes-pixie/kubernetes-integration/get-started/introduction-kubernetes-integration
           - title: Compatibility and requirements
             path: /docs/kubernetes-pixie/kubernetes-integration/get-started/kubernetes-integration-compatibility-requirements
-          - title: Release notes
-            path: /docs/release-notes/infrastructure-release-notes/kubernetes-integration-release-notes
+          - title: Changes since v3
+            path: /docs/kubernetes-pixie/kubernetes-integration/get-started/changes-since-v3
       - title: Installation
         pages:
-          - title: Install Kubernetes integration
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure
-          - title: Manual install using Helm
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm
-          - title: Kubernetes for Windows
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-windows
-          - title: Install Fargate integration
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-fargate-integration
-          - title: Configure control plane monitoring
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/configure-control-plane-monitoring
-          - title: Configure Kubernetes with a proxy
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/configure-kubernetes-proxy
-          - title: Predefined alert policy
-            path: /docs/kubernetes-pixie/kubernetes-integration/kubernetes-events/kubernetes-integration-predefined-alert-policy
-          - title: Recommended alerts
-            path: /docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-recommended-alert-policy
+        - title: Install Kubernetes integration
+          path: /docs/kubernetes-pixie/kubernetes-integration/installation/kubernetes-integration-install-configure
+        - title: Manual install using Helm
+          path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-kubernetes-integration-using-helm
+        - title: Install Fargate integration
+          path: /docs/kubernetes-pixie/kubernetes-integration/installation/install-fargate-integration
       - title: Understand and use data
         pages:
           - title: Explore your Kubernetes cluster
             path: /docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-cluster-explorer
           - title: Use your Kubernetes data
             path: /docs/kubernetes-pixie/kubernetes-integration/understand-use-data/find-use-your-kubernetes-data
+          - title: Kubernetes events
+            path: /docs/kubernetes-pixie/kubernetes-integration/understand-use-data/kubernetes-events-integration
```

Comment:
we can fix also this issue https://github.com/newrelic/nri-kubernetes/issues/255

Creating a comment not to forget about it